src/icons/icons.stories.html

```diff
@@ -0,0 +1,21 @@
+<article class="stories-icons">
+  <h2>Icons</h2>
+  <section>
+    <div>
```

Comment:
Let's add the name of the icon to the documentation, too. Maybe just a div below the svg that has the name in it.

Reply:
Ok. I've done the changes above. Looks like changes are present in branch on this repo called icons.